packages/scss/src/commons/utils/namespace.scss

```diff
@@ -0,0 +1,17 @@
+@use '@lucca-front/scss/src/commons/config';
+
+// default to 'without: rule' is namespace.$isNamespaced is false
+// default to 'with: rule' is namespace.$isNamespaced is true
+$defaultAtRoot: config.$isNamespaced and 'with: rule' or 'without: rule';
```

Comment:
Having the need to explain what the code does seems like a signal it could be simplified.

The oneliner is interesting to understand Sass specificities, but isn’t it slowing down understanding compared to the following example?

```scss
$defaultAtRoot: 'without: rule';

@if $isNamespaced {
  $defaultAtRoot: 'with: rule';
}
```